setup.py

```diff
@@ -362,13 +357,18 @@ def finalize_options(self):
 
     def run(self):
         self.run_command('bdist_app')
-        app = 'dist/%s.app' % PACKAGE
+        # Get dmgbuild
+        from utils.download import download
+        dmgbuild = download(
+          'https://github.com/morinted/dmgbuild/releases/download/v1.2.1%2Bplover/dmgbuild-1.2.1.pex',
+          '548a5c3336fd30b966060b84d86faa9a697b7f94'
+        )
+        subprocess.check_call(('chmod', '+x', dmgbuild))
         dmg = 'dist/%s.dmg' % PACKAGE
-        cmd = 'bash -x osx/app2dmg.sh %s %s' % (app, dmg)
+        cmd = '%s -s osx/dmg_resources/settings.py Plover %s' % (dmgbuild, dmg)
```

Comment:
Why not use `cmd = 'python2 %s -s osx/dmg_resources...` so you don't have to chmod dmgbuild?